app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,28 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] In the `users_controller.rb` file, please implement the `index` and `show` methods. 🙏 

- { } Following the first requirement ☝️ Please create and implement the corresponding views `erb` code as appropriate to the project's requirements. `# app/views/users/index.html.erb` && `# app/views/users/show.html.erb`